vectors/cryptography_vectors/pkcs7/rsa_ca.pem

```diff
@@ -0,0 +1,19 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Anything in the vectors/ dir needs to be documented in `test-vectors.rst`. (You can also split that out into a seperate PR so we can merge it easily).

Reply:
OK, will do!

Reply:
Opened #11843 for the matter 😄